bears/vcs/git/GitCommitBear.py

```diff
@@ -58,22 +63,38 @@ def run(self,
                 yield Result(self, "HEAD commit has no message.")
             return
 
-        yield from self.check_shortlog(shortlog_length, stdout[0])
+        yield from self.check_shortlog(shortlog_length,
+                                       shortlog_trailing_period,
+                                       stdout[0])
         yield from self.check_body(body_line_length, force_body, stdout[1:])
 
         os.chdir(old_dir)
 
-    def check_shortlog(self, shortlog_length, shortlog):
+    def check_shortlog(self,
+                       shortlog_length,
+                       shortlog_trailing_period,
+                       shortlog):
         """
         Checks the given shortlog.
 
-        :param shortlog_length: The maximum length of the shortlog. The newline
-                                character at end does not count to the length.
-        :param shortlog:        The shortlog message string.
+        :param shortlog_length:          The maximum length of the shortlog.
+                                         The newline character at end does not
+                                         count to the length.
+        :param shortlog_trailing_period: Whether a dot shall be enforced at end
+                                         end or not (or ``None`` for "don't
+                                         care").
+        :param shortlog:                 The shortlog message string.
         """
         if len(shortlog) > shortlog_length:
             yield Result(self, "Shortlog of HEAD commit is too long.")
 
+        if (shortlog[-1] != ".") == shortlog_trailing_period:
+            result_string = (
+                "Shortlog of HEAD commit contains no period at end."
+                if shortlog_trailing_period else
+                "Shortlog of HEAD commit contains a period at end.")
```

Comment:
having those three lines directly in the result constructor wouldn't be more complicated but shorter